README.md

```diff
@@ -405,6 +406,7 @@ The primary steps in kpatch-build are:
 - Link all the output objects into a cumulative object
 - Generate the patch module
 
+
 ### Patching
```

Comment:
I guess the pattern is to double-space the ### sub-sections?  If so, does the Debian 7.x Install subsection need an extra line, too?

Reply:
I tried to follow the pattern in the document and that seemed to be the most consistent way. The Debian 7.x section has four levels (####) so I added only one empty line there.